mci/api/v1_0_0/user_handler.py

```diff
@@ -349,15 +386,22 @@ def _find_address_id(self, address):
             except Exception:
                 _postal_code = None
 
+            try:
+                _county = address.get('county')
+            except Exception:
+                _county = None
```

Comment:
Line 395 assigns a default value (an empty string) and uppercases the the result. I have a couple questions:

1. Should you apply similar logic to `_county` (line 390)? 

2. If so, then you risk having both `None` and an empty string represent a non-populated/null `county` property. (Same for `country`.) Is this an issue? If so, it might make sense to do either: 
* `_county = ''` on line 392
* OR `_county = address.get('county', None)` on line 390, though this solution does not protect you from POSTing with `{"county": ""}`. 

I suppose it's a question of how you want to represent "this Address does not have a County" – with `None` or with an empty string? [(The database uses `None` as the default, fyi.)](https://github.com/brighthive/mci-database/blob/master/mci_database/db/models/models.py#L42)